plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/hosted/RecreateMetadataWalkerProcessor.java

```diff
@@ -191,9 +193,21 @@ public void onCollectionExit(final WalkerContext context, final StorageCollectio
       // do this in memory, as package.json is expected to be fairly small (few kb at most)
       final ByteArrayOutputStream bos = new ByteArrayOutputStream(Ints.checkedCast(tarEntry.getSize()));
       StreamSupport.copy(tarArchiveInputStream, bos, StreamSupport.BUFFER_SIZE);
-      final Map<String, Object> packageJson = objectMapper
-          .readValue(bos.toByteArray(), new TypeReference<HashMap<String, Object>>() { });
-      return packageJson;
+
+      try {
+        return objectMapper.readValue(bos.toByteArray(), new TypeReference<HashMap<String, Object>>() { });
+      }
+      catch (JsonParseException e) {
+        // fallback
+        if (e.getMessage().contains("Invalid UTF-8 middle byte")) {
+          log.debug("JSON UTF-8 problem for package {} : {}, attempting with charset ISO-8859-2...",
```

Comment:
reflex: ISO-8859-2 is Latin2, used in E-EU (and contains HU characters too for example)... :smile:  
Fix imminent.